frontend/src/scenes/session-recordings/player/inspector/v2/components/ItemEvent.tsx

```diff
@@ -18,6 +18,7 @@ export function ItemEvent({ item, expanded, setExpanded }: ItemEventProps): JSX.
                     <PropertyKeyInfo
                         className="font-medium"
                         disablePopover
+                        disableIcon
```

Comment:
Why no icon 😅 ? I thought it looked pretty good to indicate "native" events versus custom ones?

Reply:
i didn't feel too strongly about it so i'll add it back in :)